CONTRIBUTING.md

```diff
@@ -37,381 +48,295 @@ to sign the CLA. Therefore,
-
-If you want to notify and include other stakeholders in your PR review, use the
-`/cc` command. For example: `/cc @stakeholder_id1 @stakeholder_id2`
-
-## Docs contributor roles
-
-Because contributing to the documentation requires a different skill set than
-contributing to the Knative code base, we've defined the roles of documentation
-contributors separately from the roles of code contributors.
-
-If you're looking for code contributor roles, see
-[ROLES](https://github.com/knative/community/blob/master/ROLES.md).
-
-### Member
-
-Established contributor to the Knative docs.
+The
+[Knative Documentation Working Group](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#documentation)
+meets biweekly on Tuesdays at 9:30am PT.
+[Click here](https://calendar.google.com/calendar/embed?src=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com)
+to see the exact dates on the Knative working group calendar.
 
-Members are continuously active contributors in the community. They can have
-issues and PRs assigned to them, might participate in working group meetings,
-and pre-submit tests are automatically run for their PRs. Members are expected
-to remain active contributors to the Knative documentation.
+The Working Group meetings are used to discuss documentation strategy and
+policy; we expect individual PR review to happen mostly asynchronously via
+GitHub, and [Slack](https://slack.knative.dev).
 
-All members are encouraged to help with PR reviews, although each PR must be
-reviewed by an official [Approver](#approver). In their review, members should
-be looking for technical correctness of the documentation, adherence to the
-[style guide](https://developers.google.com/style/), good spelling and grammar
-(writing quality), intuitive organization, and strong documentation usability.
-Members should be proficient in at least one of these review areas.
+# Workflow
 
-### Requirements
+There are roughly two workflows: writing / contributing new content, and
+refining and reviewing existing content.
 
-- Has made multiple contributions to the project or community. Contributions
-  might include, but are not limited to:
+## New Content
 
-  - Authoring and reviewing PRs on GitHub in the Docs or Website repos.
+We expect most new content to be written by the subject matter expert (SME)
+which would be the contributor who actually worked on the feature or example.
 
-  - Filing and commenting on issues on GitHub.
+When writing new content, keep the following in mind:
 
-  - Contributing to working group or community discussions.
+- Write one [type of documentation](#type of documentation) per page.
 
-- Subscribed to
-  [knative-dev@googlegroups.com](https://groups.google.com/forum/#!forum/knative-dev).
+- If possible, follow the style/template of other documents of the same type.
 
-- Actively contributing to 1 or more areas.
+- Focus mostly on technical correctness; structure and language should be
```

Comment:
I do like where this is going but I dont think "technical correctness" explains enough. Ideally, the scope should target the audience (ie. the level of technical aptitude/experience would differ). In other words, we can tell users to "do this" and "run that" but if we fail to explain why, then there is a lost opportunity for learning.